backend/app/services/dingtalk_wikispace_service.py

```
@@ -0,0 +1,443 @@
+                "Fetching documents for knowledge base '%s' (workspace_id=%s)",
+                kb_name,
+                kb_id,
+            )
+
+            try:
+                await DingTalkWikiSpaceService._list_nodes_in_wikispace(
+                    docs_mcp_url=nodes_url,
+                    workspace_id=kb_id,
+                    all_nodes=all_nodes,
+                )
+            except Exception as exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)